### PR TITLE
Stepper

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

Issue Number: GOOD-225

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```
Screenshot 2024-05-21 at 10 24 05
Screenshot 2024-05-21 at 10 24 11
Screenshot 2024-05-21 at 10 24 17
Screenshot 2024-05-21 at 10 24 24

### CODE REVIEW

File: src/layout/Stepper/Step.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  markerLabel?: string;
+
+  /**
+   * CSS Classname to applied to the step
+   */
+  className?: string;
+}
+
+export const Step: FC<StepProps> = ({ children, className }) => (
+  <div className={className}>{children}</div>
```
Comment (amcdnl) [inline src/layout/Stepper/Step.tsx:16]: This probably needs a role?

File: src/layout/Stepper/Stepper.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+   * Theme for the Stepper.
+   */
+  theme?: StepperTheme;
+
+  /**
+   * Show number for every step
+   */
+  numbered?: boolean;
+}
+export const Stepper: FC<StepperProps> = ({
```
Comment (amcdnl) [inline src/layout/Stepper/Stepper.tsx:23]:
```suggestion

export const Stepper: FC<StepperProps> = ({
```

File: src/layout/Stepper/Step.tsx (hunk 1 above)
Comment (evgenoid) [reply src/layout/Stepper/Step.tsx:16]: Do you mean this one https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles?

**Review by amcdnl — APPROVED** [review]

File: src/layout/Stepper/Step.tsx (hunk 1 above)
Comment (evgenoid) [reply src/layout/Stepper/Step.tsx:16]: Probably `listitem` but I'm not sure

**Pull request merged**